### PR TITLE
Added NapsterController

### PR DESCRIPTION
**Pull request by Pfennigbaum**

### CODE REVIEW

File: code/js/modules/Sitelist.js [hunk 1]
```diff
@@ -100,6 +100,7 @@
       "music.microsoft": { name: "Microsoft Groove", url: "http://music.microsoft.com", controller: "MicrosoftController.js" },
       "mycloudplayers": { name: "My Cloud Player", url: "http://www.mycloudplayers.com" },
       "myspace": { name: "MySpace", url: "http://www.myspace.com" },
+	  "napster": { name: "Napster", url: "https://app.napster.com", controller: "NapsterController.js" },
```
Comment (berrberr) [inline code/js/modules/Sitelist.js]: Please fix alignment

File: code/js/controllers/NapsterController.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  new BaseController({
+    siteName: "Napster",
+	play: ".player-play-button .icon-play-button",
+    pause: ".player-play-button .icon-pause2",
+    playNext: ".player-advance-button",
+    playPrev: ".player-rewind-button",
+	like: ".like-button",
+	dislike: ".dislike-button",
+	buttonSwitch: true,
```
Comment (berrberr) [inline code/js/controllers/NapsterController.js]: Please fix all the key alignment in this object to maintain style

**Comment by berrberr** [comment]
Thanks! Fix the style issues and this is good.

**Comment by Pfennigbaum** [comment]
fixed alignment issues

**Comment by edwardlafoy** [comment]
Napster is live now, so the extension no longer works. Can we get this merged in?

**Comment by edwardlafoy** [comment]
@berrberr

**Comment by berrberr** [comment]
Thanks @Pfennigbaum! 👍

**Pull request merged**